packages/expo-video/ios/Records/DRMOptions.swift

```diff
@@ -10,6 +10,9 @@ internal struct DRMOptions: Record {
   @Field
   var licenseServer: String?
 
+  @Field
+  var getLicense: ((String, String, String) -> Promise<String>)?
```

Comment:
Have you tested this? As far as I remember records don't support passing functions yet 😅

Reply:
I am on Windows my self, so I couldn't find an easy way to test. But I guess I'll need to find another way for that function haha

Reply:
@tsapeta I've migrated to an event now. Although, I am not sure if the Modules API supports using data returned from an event. 😅